services/src/Microsoft.Azure.IIoT.Services.OpcUa.Publisher.Edge/src/Controllers/HeartbeatController.cs

```diff
@@ -42,7 +42,7 @@ public class HeartbeatController : ControllerBase {
                 throw new ArgumentNullException(nameof(heartbeat));
             }
             var result = await _orchestrator.SendHeartbeatAsync(
-                heartbeat.ToServiceModel());
+                heartbeat.ToServiceModel(), null);
```

Comment:
>, null [](http://example.com/codeflow?start=42&length=6)

I think we can revert this, the default parameter is already null

Reply:
reverted